docs/src/user/manual/explore/reports.md

```diff
@@ -50,9 +50,184 @@ User group is configurable from the User Table in the [Django Admin](../../../ad
-2. **Custom columns**: Configurable from the Django admin site by the site administrator. 
+2. **Custom columns**: Configurable from the Django admin site by the site administrator.
+
 See Activities section in the [Django Admin](../../../administrator/manual/django-admin.md) for the details.
 
+### Reports Page with Reports Filter
+
+#### Report Type Filter
+
+![Reports Type](./img/reports-3.png)
+
+1. **Report Type**: Report type filter with the reports type.
+
+2. **Species**: The species icon displays with the scientific name of the selected species.
+
+3. **Organisation**: The organisation icon displays with the count of selected organisations.
+
+4. **Property**: The property icon displays with the count of selected properties.
+
+5. **Clock Icon**: The clock icon serves as a visual indicator for the year range filter with year range.
+
+6. **Activity**: The activity icon displays with the count of selected activities.
+
+7. **Organisation List**: Show the name of the selected organisations.
+
+8. **Property List**: Show the name of the selected properties.
+
+9. **Activity List**: Show the name of the selected activities.
+
+     **Activity Report Filter**
+
+     ![Reports Type Filter](./img/reports-4.png)
+
+     1. **Activity Report**: Selected `Activity report`.
+
+     **Activity Report Tables**
+
+     ![Activity Reports Table](./img/reports-5.png)
+
+     **Activity Types**
+
+     The Activity Report includes information for the following activity types:
+
+     1. **Translocation (Intake)**: This table provides details on translocation activities related to the intake of species, for properties per year.
+
+     2. **Translocation (Offtake)**: This table displays information about translocation activities related to off-take,  for properties per year.
+
+     3. **Planned Hunt/Cull**: This table contains data related to planned hunting or culling activities,  for properties per year.
+
+     4. **Planned Euthanasia/DCA**: This table presents information on planned euthanasia or disease control activities, for properties per year.
+
+     5. **Unplanned/Illegal Hunting**: This table offers insights into unplanned or illegal hunting activities,  for properties per year.
+
+     **Table Columns**
+
+     Each of the tables associated with the activity types contains the following columns:
+
+     - **Property Name**: The name of the property where the activity took place.
+     - **Scientific Name**: The scientific name of the species involved in the activity.
+     - **Common Name**: The common name of the species.
+     - **Year**: The year in which the activity occurred.
+     - **Total**: The total count of the species involved in the activity.
+     - **Adult Male**: The count of adult male individuals.
+     - **Adult Female**: The count of adult female individuals.
+     - **Juvenile Male**: The count of juvenile male individuals.
+     - **Juvenile Female**: The count of juvenile female individuals.
```

Comment:
Please use `*` for unordered lists to avoid linting errors

Reply:
okay

Reply:
okay